lib/src/ffmpeg/filters/crop_filter.dart

```diff
@@ -1,20 +1,25 @@
 import 'package:ffmpeg_cli/src/ffmpeg/ffmpeg_command.dart';
 
+/// Crops the input video to given dimensions.
 class CropFilter implements Filter {
-  const CropFilter({
-    required this.width,
-    required this.height,
-  });
+  const CropFilter({required this.width, required this.height, this.x, this.y});
```

Comment:
Please format named parameters so that there's one per line, i.e., always use a trailing comma.